eos-install-flatpak-extensions

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+# Copyright (C) 2017 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+# Install flatpak extensions
+extensions=(org.freedesktop.Platform.Icontheme.EndlessOS)
+for extension in ${extensions[@]}; do
+	flatpak install eos-sdk $extension || true
```

Comment:
AFAICS this will download flatpaks at boot and block boot on network I/O, so it's a big no-no.
Can we make this part of an update to the Freedesktop runtime (or the Endless runtime)?

Reply:
We can make it autodownload alongside com.endlessm.apps.Platform//2, assuming gnome-software will respect the `autodownload = true` metadata.